### DISCUSSION TITLE
what board definition should i use for the ESP32 Dev Board Type-C USB CH340C  ?

### DISCUSSION THREAD

**Discussion by ChisWest** [espressif/arduino-esp32#7609] [Q&A]
what board definition should i use for the ESP32 Development Board Type-C USB CH340C WiFi+Bluetooth Ultra-Low Power Dual Core ESP32-DevKitC-32 ESP-WROOM-32 Expansion Board?  Im using the arduino IDE on a WIN 11 OS.  I have the ESP board manager loaded but my board does not show up in the list of boards.  this is my first attempt at using this configuration as well as the ESP32.  any help will be greatly appreciated.

**Comment by mrengineer7777** [comment]
That looks like a standard ESP32-V3 module (not the C, S, H versions).  Do you have an option for an Adafruit featheresp32 or other generic ESP32?
